releases/release-4.0.2.md

```
@@ -13,48 +13,48 @@ TiDB バージョン: 4.0.2
+    -   `REGEXP()`関数照合順序をサポート [#17581](https://github.com/pingcap/tidb/pull/17581)
+    -   `sql_select_limit`セッションとグローバル変数サポート [#17604](https://github.com/pingcap/tidb/pull/17604)
+    -   新しく追加されたパーティションのリージョン分割をデフォルトでサポート[#17665](https://github.com/pingcap/tidb/pull/17665)
+    -   `IF()` / `BITXOR()` / `BITNEG()` / `JSON_LENGTH()`関数をTiFlashコプロセッサー にプッシュすることをサポート [#17592](https://github.com/pingcap/tidb/pull/17592) [#17651](https://github.com/pingcap/tidb/pull/17651)
+    -   `COUNT(DISTINCT)` のおおよその結果を計算する新しい集計関数`APPROX_COUNT_DISTINCT()`サポートします。 [#18120](https://github.com/pingcap/tidb/pull/18120)
+    -   TiFlashでの照合順序をサポートし、照合関連の関数をTiFlash にプッシュします。 [#17705](https://github.com/pingcap/tidb/pull/17705)
+    -   `INFORMATION_SCHEMA.INSPECTION_RESULT`テーブルに`STATUS_ADDRESS`列を追加して、サーバのステータス アドレスを示します。 [#17695](https://github.com/pingcap/tidb/pull/17695)
+    -   `MYSQL.BIND_INFO`表に`SOURCE`列を追加して、バインディングの作成方法を示します[#17587](https://github.com/pingcap/tidb/pull/17587)
+    -   SQL文のプランキャッシュの使用状況を示すために、 `PERFORMANCE_SCHEMA.EVENTS_STATEMENTS_SUMMARY_BY_DIGEST`表に`PLAN_IN_CACHE`と`PLAN_CACHE_HITS`列を追加します。 [#17493](https://github.com/pingcap/tidb/pull/17493)
+    -   `enable-collect-execution-info`構成項目と`tidb_enable_collect_execution_info`セッション変数を追加して、各演算子の実行情報を収集し、その情報をスロークエリログ に記録するかどうかを制御します。 [#18072](https://github.com/pingcap/tidb/pull/18072) [#18073](https://github.com/pingcap/tidb/pull/18073)
+    -   スロークエリログでクエリの感度を下げるかどうかを制御するグローバル変数`tidb_slow_log_masking`追加します。 [#17694](https://github.com/pingcap/tidb/pull/17694)
+    -   `storage.block-cache.capacity` TiKV構成項目[#17671](https://github.com/pingcap/tidb/pull/17671) `INFORMATION_SCHEMA.INSPECTION_RESULT`テーブルに診断ルールを追加します。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)